backend/lcfs/db/models/transaction/Transaction.py

```diff
@@ -6,6 +6,18 @@
 
 from lcfs.db.base import BaseModel, Auditable, EffectiveDates
 
+transaction_type_to_id_prefix_map = {
```

Comment:
I would like to move these into the /utils/constants.py file. Transactions are not directly related to entities on the backend so better to keep these references separate. This is one of those annoying moments where Transactions on the backend don't necessarily mean the same thing as on the frontend.

Reply:
Done! Moved to constants.py. Wasn't sure if I should make it a class / uppercase it so let me know.